src/Composer/Installers/EzPlatformInstaller.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Composer\Installers;
+
+class EzPlatformInstaller extends BaseInstaller
+{
+    protected $locations = array(
+        'meta-assets' => 'web/assets/ezplatform/',
```

Comment:
Keep in mind an update will delete that entire directory.  So there can only ever be one of that type and it will not play well with the assets type.  Perhaps you should emit a warning of some kind.